modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
++
+Example: v1  
+
+|===
+
+=== Asset Versioning
+
+When you publish a version of an asset that already exists in Exchange, the patch version is incremented by default. However, you can set the version using semantic versioning notation `x.x.x` for major, minor, and patch versions respectively. 
+
+Use the following symbols before the version nodes to control how an asset version is incremented:
+
+[%header,cols="15a, 70a"]
+|===
+| Symbol
+
+| Version Node
+
+| ^ | patch
+
+| ~ | minor
+
+| & | major 
+
+Because '&' is a reserved character in YAML, a semantic versioning string using this character must be enclosed in a single quote. For example: '&x.x.x'
+
+|===
+
+==== Examples of Asset Versioning
+
+The following table shows how the asset version is incremented for example semantic versioning strings. 
+
+[%header,cols="15a, 70a"]
+|===
+| Asset Version (versioning) 
+
+| Explanation
+
+| ^x.x.x (Default) | Search for the latest version and increment the patch
+
+| ^1.x.x | Search for the latest version of major 1 and increment the patch
+
+| ~1.x.x| Search for the latest version of major 1 and increment the minor
+
+| ^1.0.x | Search for the latest version of major 1, minor 0 and increment the patch
+
+| ^3.x.x | Search for the latest version of major 3 and increment the patch
+
+| ^3.x.x | Search for the latest version of major 3 and increment the patch
+
+| '&x.x.x' | Increment to the next the major version. 
+
+|===
+
+For more information on asset versioning in Exchange, see xref:to-change-raml-version.adoc[Change the Version of an API Asset].
+
+=== Example Descriptor File
+
+Following is an example descriptor file that describes the cataloging information for two APIs. 
+
+[source,yaml]
+----
+
+ #%Catalog Descriptor 0.6 # <1>
+triggerConditions: # <2>
+  branches:
+    - main
+    - release/(.*)
+  tags:
+    - support
+    - release/(.*)
+  user:
+    - admin
+contact: # <3>
+  name: 'John Doe'
+  email: 'john.doe@org.com'
+apis: # <4>
+  - spec: api-spec/codat.json
+    assetId: codat-api
+    docs:
+      add: api-spec/add.md
+      home: home.md
+    customFields:
+      custom: value
+      another: field
+    tags:
+      - codat
+      - gcp
+    versioning: 2.0.x
+    apiVersion: v3
+  - spec: api-spec/billing-api.json
+    assetId: my-awesome-api
+    tags:
+      - finance
+      - aws
+    categories:
+      API Type:
+        - System API
+        - Experience API
+      Organization:
+        - Finance
+        - Billing
+    versioning: ~1.x.x
+    apiVersion: v1
+----
+<1> Provides the start line for the descriptor file
+<2> Sets trigger conditions
+<3> Provides the contact name and email ID 
+<4> Specifies the API information to be published
+
+== Configure Command Autocompletion
```

Comment:
Another Level-2 not accounted for in intro

Reply:
Yes, I added this at the end, and was originally just covering it in the note on line 50. This is not one of the tasks that a customer needs to perform to automate the publishing of their assets. It is optional and is really part of the installation/configuration -- a one-time task. Again, it would bog down the "getting started" to have a lot of details on something that is more like configuring help. It might be more useful later if we have more options, but we already have the help. It is also a plugin from oclif, and we weren't able to change the built-in help, which isn't great. I wasn't sure of the license we have to document this, but since it seems to be open-source software, I couldn't find end-user doc for the feature either. Something to discuss when we talk.